docs/pages/guide/02-start.md

```diff
@@ -0,0 +1,409 @@
+on models takes you, in depth, through the step to step instruction on
+writing your Elide data model and testing it appropriately. Yavin
+leverages the Elide HJSON configuration language to define the semantic
+model.  For complete documentation on the language, visit <span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23model-configuration&amp;sa=D&amp;ust=1606228594029000&amp;usg=AOvVaw0bWaOa678DAXH4CB2iucYe" class="c14">https://elide.io/pages/guide/v5/04-analytics.html#model-configuration</a></span>.
+ For a getting started tutorial, visit <span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23example-configuration&amp;sa=D&amp;ust=1606228594029000&amp;usg=AOvVaw2kw5ry4iUdA4oVVcADetKl" class="c14">https://elide.io/pages/guide/v5/04-analytics.html#example-configuration</a></span><span
+class="c11">.</span>
+
+<span class="c11"></span>
+
+<span class="c11">Yavin comes bundled with example Netflix data for
+movies and television shows.  The following sections will illustrate how
+Yavin is configured to explore this data.</span>
+
+<span class="c11"></span>
+
+#### <span class="c37 c36">Our example use case</span>
+
+<span class="c11">Analytics on Netflix movie and TV shows titles</span>
+
+#### <span class="c37 c36">Connections, Tables, Dimensions, Measures and Joins</span>
+
+Your connection can be set up as: <span class="c34">dialect: H2</span>
+
+```
+{
+  dbconfigs: [
+    {
+      name: DemoConnection
+      url: jdbc:h2:mem:DemoDB;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'classpath:create-demo-data.sql'
+      driver: org.h2.Driver
+      user: guest
+      dialect: H2
+    }
+  ]
+}
+
+```
+
+
+Defining your Table/s: netflix_titles
+
+```
+name:  NetflixTitles
+table: netflix_titles
+```
+
+Defining your Dimensions title_id, show_type, title  (What are Dimensions?)
+
+```
+  dimensions: [
+        {
+          name: title_id
+          type: TEXT
+          definition: '{{title_id}}'
+        }
+        {
+          name: show_type
+          type: TEXT
+          definition: '{{type}}'
+        }
+......
+```
+
+Defining your Measures/Metrics count, total_seasons, movie_duration  (What are Measures/Metrics?)
+
+```
+measures: [
+        {
+          name: count
+          type: INTEGER
+          definition: 'count({{title_id}})'
+        }
+        {
+          name: total_seasons
+          type: INTEGER
+          definition: "sum(cast (case when {{duration}} like '% Seasons' then REPLACE({{duration}}, ' Seasons', '') else '0' end AS INT))"
+        }
+        {
+          name: movie_duration
+          type: INTEGER
+          definition: "sum(cast (case when {{duration}} like '% min' then REPLACE({{duration}}, ' min', '') else '0' end AS INT))"
+        }
+```
+
+Defining your Joins NA for this demo. Since there is no joins with the "Netflix data set", this is jus Sample on how it can be :
```

Comment:
We should either leave this out or make sure our example data set has at least one join.